### ISSUE TITLE
[GLPI 10.0.0] [FC 2.13.0-b1] Counters dashboard empty

### ISSUE THREAD

**Issue by jcovillers78114** [pluginsGLPI/formcreator#2727]
In the simplified support interface, the dashboard counters are empty even though there is at least one open ticket for the user.

![2022-04-27 10_14_30-Catalogue de service - GLPI](https://user-images.githubusercontent.com/57490677/165473413-deccd932-d7f1-4830-b1cd-d6a365db072f.png)

**Comment by btry** [comment]
Do you have errors in php-errors or sql-errors ?

**Comment by btry** [comment]
If you upgraded from a previous Alpha version run the following SQL query

```sql
DELETE
FROM glpi_dashboards_items
WHERE dashboards_dashboards_id = (
	SELECT id
	FROM glpi_dashboards_dashboards
	WHERE `key`='plugin_formcreator_issue_counters'
);

DELETE FROM glpi_dashboards_dashboards WHERE `key`='plugin_formcreator_issue_counters';
```

Then run a forced upgrade again, as described in the releases notes of 2.13.0-beta.1

**Comment by jcovillers78114** [comment]
Yep from alpha 4.

I've got this warning in php log file:

[2022-04-27 10:02:28] glpiphplog.WARNING:   *** PHP Warning (2): count(): Parameter must be an array or an object that implements Countable in /glpi/plugins/formcreator/inc/conditionnabletrait.class.php at line 129
  Backtrace :
  plugins/formcreator/inc/form.class.php:1090        PluginFormcreatorForm->updateConditions()
  src/CommonDBTM.php:1667                            PluginFormcreatorForm->post_updateItem()
  plugins/formcreator/inc/form.class.php:1336        CommonDBTM->update()
  plugins/formcreator/front/form.form.php:143        PluginFormcreatorForm->increaseUsageCount()

**Comment by btry** [comment]
I missed to mention those instructions in the release notes. I'm updating them right now

**Comment by jcovillers78114** [comment]
The sql query made the counters disappear from the page. I can't display them anymore even if I enable and disable the option in the administration tab > entity > form > counters dashboard

**Comment by btry** [comment]
Did you run the forced upgrade ? This is mandatory to rebuild the up to date counters.

**Comment by jcovillers78114** [comment]
Ok it works, but there is no label on it. Is this normal?

![2022-04-27 10_36_16-Catalogue de service - GLPI](https://user-images.githubusercontent.com/57490677/165477717-7eb44a5d-c1fa-46b8-9499-a4b9931f2c35.png)

**Comment by btry** [comment]
Yes, there is no label. This height of the dashboard must be limited and this makes labels too small to be easily readable. We choosed then to rely on the color of the wdgets. They match the equivalent status available for tickets.

**Comment by jcovillers78114** [comment]
Ok Thx
